app/pom.xml

```diff
@@ -161,6 +165,12 @@
             <artifactId>graphql-java</artifactId>
         </dependency>
 
+        <!-- ElasticSearch -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-elasticsearch-rest-high-level-client</artifactId>
+        </dependency>
```

Comment:
I would not expect to see Elasticsearch deps in app/pom.xml.  These should be isolated in `elasticsql` module I think.  Unless I'm missing something (possible as I haven't fully grokked this PR yet) :)

Reply:
this is related to local tests, (WIP effect)